packages/dialtone-css/lib/build/less/components/item-layout.less

```diff
@@ -13,41 +13,79 @@
 //  $   BASE STYLE
 //  ----------------------------------------------------------------------------
 .dt-item-layout {
-  display: flex;
-  align-items: center;
-  min-height: calc(var(--dt-size-550) + var(--dt-size-300));
-  padding: var(--dt-space-300) var(--dt-space-400);
-  font-size: var(--dt-font-size-200);
-  line-height: var(--dt-font-line-height-300);
+  display: grid;
+  grid-template-areas:
+    'left content right selected';
+  grid-template-columns: repeat(4, minmax(0, max-content));
 
-  &--content {
-    flex-grow: 1;
-    min-width: var(--dt-size-0);
-  }
+  &:not(&--custom) {
```

Comment:
This is the key change.
I would mention it in the component explanation (`.mdx`)

Reply:
This is mentioned [here](https://github.com/dialpad/dialtone/pull/160/files#diff-09cfc539bcbe93dff729532458a0a55b49605cc166a70ceab01c16c00b4bb1deR37-R44) or what else would you add to this?

Reply:
I don't really think that it's necessary to mention the "how" here, mentioning that adding the prop will remove the styling should be enough, but I don't have problem adding it, so I'll add this to the docs.

Reply:
We are adding a class to remove styles from another class, but the other class is still there.
Normally you add a class to add styles, in this case, the class is removing styles and _not directly_(!).

You can have the `unstyle` prop true, but if you manage to remove the class `dt-item-layout--custom` (which is not adding any styles) you will add styles (from `dt-item-layout` which is not adding styles initially cos `unstyle` was `true`)

This behaviour can be confusing, so to make it more clear Im suggesting this minor update.
Expose that we are adding a `--custom` class that it need to be there to trigger different styles from `dt-item-layout` class.

Hope this can convince a little bit more my suggestion.